modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,127 @@
+== Fedora CoreOS - Getting Started
+
+Fedora CoreOS has no install-time configuration. All Fedora CoreOS installs or cloud images start the same, then on first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+
+=== Launching Fedora CoreOS
+
+==== Booting on AWS:
+
+To boot on AWS, find the correct ami in the https://builds.coreos.fedoraproject.org/browser[release browser] and specify the Ignition config as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. Note that while the AWS documentation mentions cloud-init and scripts, Fedora CoreOS only accepts Ignition configs; it does not support cloud-init or running scripts from userdata.
```

Comment:
s/ami/AMI/

We should probably link to the download page instead of the release browser, even if it doesn't exist yet.

Reply:
Do we know what the URL will be?

Reply:
:+1: the URL will be https://getfedora.org/coreos/download/